### DISCUSSION TITLE
[podman]: SSL_ERROR_SYSCALL Connecting to Server

### DISCUSSION THREAD

**Discussion by adambkaplan** [itzg/docker-minecraft-server#2665] [Q&A] [answered]
When running the latest image with Podman on Fedora 39, I get the following errors trying to curl the server's port:

```sh
$ curl --insecure https://localhost:25565
curl: (35) OpenSSL SSL_connect: SSL_ERROR_SYSCALL in connection to localhost:25565 
```
Accessing via browser results in all sorts of fun TLS/SSL errors (and no page)
On the container side, I see no access logs in stdout indicating that I'm trying to make a connection.

Any idea what is going on here?

Relevant version info:

- Image: latest (digest 6c78aff9969f, published 3 days ago)
- Podman:
  ```
  Client:       Podman Engine
  Version:      4.8.3
  API Version:  4.8.3
  Go Version:   go1.21.5
  Built:        Wed Jan  3 09:11:40 2024
  OS/Arch:      linux/amd64
  ```
- OS: Fedora 39

**Comment by itzg** [comment, marked as answer]
The [Minecraft protocol](https://wiki.vg/Protocol) is a proprietary TCP protocol and **not HTTP**. You cannot curl to it.

**Comment by adambkaplan** [comment]
This is probably a noob question then - I assume to connect to the server you need to fire up the Minecraft game and point it to the correct server?

**Reply by itzg** [reply]
Yes you do, but curl is not a Minecraft client. Curl is for accessing web servers. 

If you're wanting a command line tool to simply check for connectivity then you can use https://github.com/itzg/mc-monitor

**Reply by itzg** [reply]
To also clarify, the server is not the full game. You will need to buy a copy of Minecraft that is both standalone and can connect to Minecraft servers.

**Reply by adambkaplan** [reply]
I'm so used to things having a web console for admin. I'll close this discussion out and keep playing around with the Minecraft game itself (and defer to the plethora of tutorials online on running your own server).
